### PR TITLE
feat: exclude all *.lock files

### PR DESCRIPTION
**Pull request by spenserblack**
This registers more files to be excluded for these types of projects (specifically, their lock files):
- Python
- Ruby
- Rust

### CODE REVIEW

File: src/utils/git.ts [hunk 1]
```diff
@@ -9,9 +9,17 @@ export const assertGitRepo = async () => {
...
 	'package-lock.json',
 	'yarn.lock',
 	'pnpm-lock.yaml',
+	// Ruby Project Files
+	'Gemfile.lock',
+	// Python Project Files
+	'Pipfile.lock',
+	'poetry.lock',
+	// Rust Project Files
+	'Cargo.lock',
```
Comment (privatenumber) [inline src/utils/git.ts:22]: Can you change it to `*.lock` and remove `yarn.lock` too?

Comment (spenserblack) [reply src/utils/git.ts]: Done :slightly_smiling_face: https://github.com/Nutlope/aicommits/pull/108/commits/b893d4ad5f6a7b5c240941127fd1e5cd59d62267

Did a quick search of `path:*.lock` in GitHub to see if there might be any type of file with this extension that should *not* be excluded, but it looks like there aren't (or they're exceptionally rare).

Comment (spenserblack) [reply src/utils/git.ts]: Side note: This makes me think that there should be some way to change the list of excluded files (config file and/or CLI option), for the rare situation that someone uses a non-standard filename or file extension.

Comment (privatenumber) [reply src/utils/git.ts]: True. Happy to accept a PR for a way to exclude patterns in the config file.

**Pull request merged**